de/figures/figure4.py

```diff
@@ -13,3 +13,5 @@ def makeFigure():
     subplotLabel(ax)
 
     return f
+
+
```

Comment:
You should only have 1 white line after the end of the last function in a file. (This is a linting problem)
Please remove one of the lines.